ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2374,3 +2374,44 @@ def test_torch_sgn(
         input=input[0],
         out=None,
     )
+
+
+@handle_frontend_test(
+    fn_tree="torch.nan_to_num",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

Please use valid dtypes instead of strictly using floats

Reply:
Hi @ZiadAmerr, Thank you for the review! I saw that valid dtype includes complex dtypes which are not inherently supported for the API. 

Further, I took reference from the **nan_to_num** API  for other frontends (numpy, JAX), which have used **float** dtype for testing (probably due to the fact that complex types are not supported). Also, the corresponding experimental API seems to use float.

Reply:
I will look into that and get back to you as soon as possible :)
But feel free to take a look at `get_dtypes(...)` documentation as follows:

> Draws a valid dtypes for the test function. For frontend tests, it draws the datatypes from the intersection between backend framework data types and frontend framework dtypes, otherwise, draws it from backend framework data types.